editor/debugger/editor_debugger_tree.cpp

```diff
@@ -187,6 +189,27 @@ void EditorDebuggerTree::update_scene_tree(const SceneDebuggerTree *p_tree, int
 			}
 		}
 
+		if (!node.extra_data.is_empty()) {
+			// Add buttons.
+			if (node.extra_data.has("scene_file_path")) {
+				String node_scene_file_path = node.extra_data["scene_file_path"];
+				Ref<Texture2D> button_icon = get_theme_icon(SNAME("InstanceOptions"), SNAME("EditorIcons"));
+				// WARNING: If this tooltip doesn't have the file path, the Scene opening won't work!
```

Comment:
The comment can be simply informational, no need to make it a warning.
Also "Scene" shouldn't be capitalized.

Reply:
In retrospective, because **TreeItems** inherit Object, I could store the path in the Object metadata, which is very similar to what I did in #66665.